content/glance.pl.md

```diff
@@ -0,0 +1,129 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:"
+
+title: "ATAG w skrócie"
+nav_title: "ATAG w skrócie"
+lang: pl   # Change "en" to the translated language shortcode
+last_updated: 2024-03-05   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+translators:
+  - name: "Stefan Wajda"
+
+
+github:
+  repository: w3c/wai-intro-atag
+  path: content/glance.pl.md   # Add the language shortcode to the middle of the filename, for example: content/glance.fr.md
+  
+permalink: /standards-guidelines/atag/glance/pl   # Add the language shortcode to the end, with no slash at end, for example: /standards-guidelines/atag/glance/fr
+
```

Comment:
@zwiastunsw The `ref:` variable is missing. It is necessary to display the Polish translation.